core/src/test/scala/unit/kafka/server/TopicIdWithOldInterBrokerProtocolTest.scala

```diff
@@ -94,7 +94,8 @@ class TopicIdWithOldInterBrokerProtocolTest extends BaseRequestTest {
         )).setTimeoutMs(timeout)).build()
     val response = sendDeleteTopicsRequest(request)
     val error = response.errorCounts.asScala
-    assertEquals(2, error(Errors.UNSUPPORTED_VERSION))
+    // Previous versions of the code returned UNSUPPORTED_VERSION
```

Comment:
I think we can remove this comment because `Previous versions of the code` is a little ambiguous.

Reply:
I could specify 2.8 here, but maybe it makes more sense to remove.